rq/worker.py

```diff
@@ -374,6 +375,11 @@ def _get_state(self):
 
     state = property(_get_state, _set_state)
 
+    def set_current_job_working_time(self, current_job_working_time, pipeline=None):
```

Comment:
I initially thought that this can be a simple variable tracked by the worker itself. Thanks for persisting this in Redis, this is also a useful information to have.

Reply:
👍